testing/build-scripts/build-sidecar.sh

```diff
@@ -27,7 +27,7 @@ if [ $STATUS -ne 0 ]; then
     exit $STATUS
 fi
 
-make -C $MAKEDIR dockerize.sidecar SIDECAR_IMAGE_NAME=amalgam8/a8-sidecar:alpine SIDECAR_DOCKERFILE=$MAKEDIR/docker/Dockerfile.sidecar.alpine
+make -C $MAKEDIR dockerize.sidecar SIDECAR_IMAGE_NAME=amalgam8/a8-sidecar:alpine SIDECAR_DOCKERFILE=$MAKEDIR/docker/Dockerfile.sidecar.envoy
 STATUS=$?
```

Comment:
we can't build alpine based images in this case. Envoy's image is based on ubuntu I think. We could just use the a8-sidecar:latest